lib/floe/workflow/state.rb

```diff
@@ -58,7 +58,7 @@ def start(_input)
         context.state["Guid"]            = SecureRandom.uuid
         context.state["EnteredTime"]     = start_time
 
-        logger.info("Running state: [#{context.state_name}] with input [#{context.input}]...")
+        logger.info("Running state: [#{self.class.name.split(":").last}:#{context.state_name}] with input [#{context.input}]...")
```

Comment:
For the state_type:state_name change...can we move that duplicated code into a helper method? (ie.. the `#{self.class.name.split(":").last}:#{context.state_name}` is duplicated here and in the finish method.  I could also see subclasses wanting to use this log prefix but in their logging, so having it available to subclasses is a bonus.